src/lib/index.ts

```diff
@@ -5,3 +5,4 @@ export {Co2js} from './co2js';
 export {TeadsAWS} from './teads-aws';
 export {TeadsCurve} from './teads-curve';
 export {WattTimeGridEmissions} from './watt-time';
+export {KeplerPlugin} from './kepler-importer';
```

Comment:
```suggestion
export {Kepler} from './kepler';
```

For naming I think we should align with the other plugins.

So just use Kepler for the name and drop importer since only Azure has it. WDYT?

Reply:
I think your naming proposition is good. If we change this we will also need to update the manifest file.

Reply:
I think this is a "nice to have" so I created #7 for it.

The current naming is good and fine for our submission 👍